src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CodeFixReference.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Refactoring.V2
+{
+    [Shared]
+    [Export(typeof(CodeFixesForProjects))]
+    public class CodeFixesForProjects
```

Comment:
Anyone know better solution for this? It feels like analyzers have first class citizen support per project basis by roslyn (have analyzerReference etc mechanisms) and code fixes instead don't have that kind of support available.